### PR TITLE
Merge 4.4  into 4.5

### PR DESCRIPTION
**Pull request by DFolchA**
|Related issue|
|---|
|Closes https://github.com/wazuh/wazuh-packages/issues/2185|

## Description

This PR merges the latest changes in 4.3 into branch 4.4.

### CODE REVIEW

**Review by teddytpc1 — APPROVED** [review]

File: debs/SPECS/wazuh-agent/debian/changelog [hunk 1]
```diff
@@ -21,6 +21,12 @@ wazuh-agent (4.4.0-RELEASE) stable; urgency=low
   * More info: https://documentation.wazuh.com/current/release-notes/
 
  -- Wazuh, Inc <info@wazuh.com>  Wed, 18 Jan 2023 12:31:50 +0000
+ 
+wazuh-agent (4.3.11-RELEASE) stable; urgency=low
+
+  * More info: https://documentation.wazuh.com/current/release-notes/
+
+ -- Wazuh, Inc <info@wazuh.com>  Thu, 08 Dec 2022 15:00:00 +0000
```
Comment (rauldpm) [inline debs/SPECS/wazuh-agent/debian/changelog:29]: This version was released after `4.4.1` version not `4.3.10`, see `4.2.7` and `4.3.1`, dates are not correct

File: debs/SPECS/wazuh-manager/debian/changelog [hunk 2]
```diff
@@ -21,6 +21,12 @@ wazuh-manager (4.4.0-RELEASE) stable; urgency=low
   * More info: https://documentation.wazuh.com/current/release-notes/
 
  -- Wazuh, Inc <info@wazuh.com>  Wed, 18 Jan 2023 12:31:50 +0000
+ 
+wazuh-manager (4.3.11-RELEASE) stable; urgency=low
+
+  * More info: https://documentation.wazuh.com/current/release-notes/
+
+ -- Wazuh, Inc <info@wazuh.com>  Thu, 08 Dec 2022 15:00:00 +0000
```
Comment (rauldpm) [inline debs/SPECS/wazuh-manager/debian/changelog:29]: Same

File: CHANGELOG.md [hunk 3]
```diff
@@ -17,6 +17,10 @@ All notable changes to this project will be documented in this file.
 
 - https://github.com/wazuh/wazuh-packages/releases/tag/v4.4.0
 
+## [v4.3.11]
+
+- https://github.com/wazuh/wazuh-packages/releases/tag/v4.3.11
+
```
Comment (rauldpm) [inline CHANGELOG.md:23]: This should be after `4.4.1`, like `4.2.7` and `4.3.1` (not 4.3.2)

File: rpms/SPECS/wazuh-agent.spec [hunk 4]
```diff
@@ -610,6 +608,8 @@ rm -fr %{buildroot}
 - More info: https://documentation.wazuh.com/current/release-notes/
 * Wed Jan 18 2023 support <info@wazuh.com> - 4.4.0
 - More info: https://documentation.wazuh.com/current/release-notes/
+* Thu Dec 08 2022 support <info@wazuh.com> - 4.3.11
+- More info: https://documentation.wazuh.com/current/release-notes/
 * Thu Nov 10 2022 support <info@wazuh.com> - 4.3.10
 - More info: https://documentation.wazuh.com/current/release-notes/
```
Comment (rauldpm) [inline rpms/SPECS/wazuh-agent.spec:614]: Same, wrong order

```
* Mon May 30 2022 support <info@wazuh.com> - 4.3.2
- More info: https://documentation.wazuh.com/current/release-notes/
* Mon May 30 2022 support <info@wazuh.com> - 3.13.4
- More info: https://documentation.wazuh.com/current/release-notes/
* Sun May 29 2022 support <info@wazuh.com> - 4.2.7
- More info: https://documentation.wazuh.com/current/release-notes/
* Wed May 18 2022 support <info@wazuh.com> - 4.3.1
- More info: https://documentation.wazuh.com/current/release-notes/
```

File: rpms/SPECS/wazuh-manager.spec [hunk 5]
```diff
@@ -830,6 +831,8 @@ rm -fr %{buildroot}
 - More info: https://documentation.wazuh.com/current/release-notes/
 * Wed Jan 18 2023 support <info@wazuh.com> - 4.4.0
 - More info: https://documentation.wazuh.com/current/release-notes/
+* Thu Dec 08 2022 support <info@wazuh.com> - 4.3.11
+- More info: https://documentation.wazuh.com/current/release-notes/
 * Thu Nov 10 2022 support <info@wazuh.com> - 4.3.10
```
Comment (rauldpm) [inline rpms/SPECS/wazuh-manager.spec:836]: Wrong order

**Review by rauldpm — CHANGES_REQUESTED** [review]
We need to check 4.3 and 4.4 branch changelogs

**Review by rauldpm — APPROVED** [review]

**Review by teddytpc1 — APPROVED** [review]

File: debs/SPECS/wazuh-agent/debian/changelog [hunk 6]
```diff
@@ -21,7 +27,7 @@ wazuh-agent (4.4.0-RELEASE) stable; urgency=low
   * More info: https://documentation.wazuh.com/current/release-notes/
 
  -- Wazuh, Inc <info@wazuh.com>  Wed, 18 Jan 2023 12:31:50 +0000
-
+
```
Comment (c-bordon) [inline debs/SPECS/wazuh-agent/debian/changelog:30]: Remove this space

**Review by c-bordon — CHANGES_REQUESTED** [review]
GJ, one minor change

File: debs/SPECS/wazuh-agent/debian/changelog (hunk 6 above)
Comment (rauldpm) [reply debs/SPECS/wazuh-agent/debian/changelog]: Done in https://github.com/wazuh/wazuh-packages/pull/2187/commits/ba90ffbca3b797b495490c5e0e83e004d6b5c304

**Review by fcaffieri — APPROVED** [review]
LGTM

**Review by c-bordon — APPROVED** [review]
LGTM

**Pull request merged**